### ISSUE TITLE
New rule: always require parens in arrow functions

### ISSUE THREAD

**Issue by spadgos** [eslint/eslint#2689]
A suggested lint rule: when using es6 arrow functions, require that the arguments list is always surrounded by parentheses. When there's only one argument the parens are optional but that introduces an inconsistent style which may be unwanted by developers.

With this rule enabled...

```js
// error
const fn = x => x;

// not an error
const fn = (x) => x;
```

**Comment by KevinGrandon** [comment]
There is a plugin for doing what you want: https://github.com/Jxck/eslint-plugin-arrow-function

And there is also a plugin for doing the exact opposite :) https://github.com/KevinGrandon/eslint-plugins/blob/master/lib/rules/arrow-function-parens.js

**Comment by gyandeeps** [comment]
Here is the perfect example of awesomeness of plugins. Now we dont have to build a single rule to accommodate both scenarios. people can just choose the correct plugin based on their need.
:100:

**Comment by spadgos** [comment]
Good enough for me! Thanks all.
